test/test_plugin_manager.rb

```diff
@@ -111,6 +111,7 @@ def with_no_gemfile
 
   context "plugins_dir is set to a different dir" do
     should "expand plugin path" do
+      skip "Jekyll doesn't support setting plugins_dir to a different dir on Windows"
```

Comment:
Note: Eventually I plan on implementing Pathutil here and making Pathutil cleaner so it can support this for Windows users.  I've a lot of Windows users here that use Jekyll so it is on my plate but since I'm a systems guy by job, I don't find much time to work on anything more than scaling Jekyll.